admin/class-acc_user_importer-admin.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 
+$logstr = "";		//handy global to store log string
```

Comment:
Yeah, I know it is not super clean. My goal is to enhance the code readability. There used to be 2 different statements for every log. Now there is only one, called log_dual.  I wanted to keep the invocation simple and did not want to pass the string back and forth. This is why I used the global.  I originally wanted to put it underneath the 'class acc_user_importer_Admin', but somehow it did not work. Forgive my lack of expertise with PHP and object oriented programming.  For now I will rename the variable as you suggested, and if you know how to use a class property, please go ahead and do that (since I am not sure what you are referring to).

Reply:
>  So tell me, is the current scope just for the file, or it is global across all wordpress installation?
That is a good question but I am not 100% sure. My understanding is that it is for the full Wordpress installation. This is true for sure for everything within `$GLOBAL` since that is a [suberglobal](https://www.php.net/manual/en/language.variables.superglobals.php). 
For the var itself, it is a [single scope](https://www.php.net/manual/en/language.variables.scope.php) which is limited, but Wordpress does some interesting things in how it loads files and plugins, so I am not sure how widely it is accessible.